src/lib/queries/staking.ts

```diff
@@ -0,0 +1,26 @@
+import { createRpc, QueryClient } from '@cosmjs/stargate';
+
+import { QueryClientImpl } from '../../codec/cosmos/staking/v1beta1/query';
+import { Params } from '../../codec/cosmos/staking/v1beta1/staking';
+
+export interface StakingExtension {
+  readonly staking: {
+    readonly params: () => Promise<Params | undefined>;
+  };
+}
+
+export function setupStakingExtension(base: QueryClient): StakingExtension {
```

Comment:
We already export a setup function and extension type from `@cosmjs/stargate` if you want to use that. It's not as lightweight, obviously.